wazo_webhookd/plugins/subscription/api.yml

```diff
@@ -177,6 +177,21 @@ paths:
           $ref: '#/responses/NotFoundError'
         '503':
           $ref: '#/responses/AnotherServiceUnavailable'
+  /subscriptions/{subscription_uuid}/hook-log:
```

Comment:
You used `hook-log` here and `hook-logs` in the implementation. I prefer the plural form.

Reply:
I have used `logs` everywhere.